app/views/schedule/scheduled_publishing_datetime.html.erb

```diff
@@ -1,49 +1,53 @@
 <% content_for :back_link, render_back_link(href: document_path(@edition.document)) %>
 <% content_for :title, t("schedule.scheduled_publishing_datetime.title") %>
-
-<% current_scheduled_publish_datetime = @edition.revision.scheduled_publishing_datetime %>
-<% default_date = Time.zone.now.tomorrow %>
-<% publish_date = current_scheduled_publish_datetime || default_date %>
-<% schedule_param = params["schedule"] ? params["schedule"] : {} %>
-<% selected_time = schedule_param["time"] ||  current_scheduled_publish_datetime&.strftime("%l:%M%P")&.strip %>
-<% action_not_selected = @issues&.select { |i| i.issue_key == :action_not_selected }&.first&.to_item %>
-
+<%
+  current_scheduled_publish_datetime = @edition.scheduled_publishing_datetime
+  default_time = Time.current.tomorrow
+  publish_time = current_scheduled_publish_datetime || default_time
+  action_not_selected = @issues&.select { |i| i.issue_key == :action_not_selected }&.first&.to_item
```

Comment:
Looks like it's not part of this PR, but the requirements system has a `items_for` method especially for this :-).

Reply:
Yup I think I squash this later in this PR